src/Aspire.Dashboard/Components/Controls/ResourceSelect.razor.cs

```diff
@@ -32,6 +32,11 @@ public partial class ResourceSelect
 
     private FluentSelect<SelectViewModel<ResourceTypeDetails>>? _resourceSelectComponent;
 
+    private static void ValuedChanged(string value)
+    {
+        // Do nothing. Required for bunit change to trigger SelectedOptionChanged.
+    }
```

Comment:
@vnbaaij I'm confused why this is needed. It's required for the other change event to trigger via:

```cs
        var resourceSelect = cut.FindComponent<ResourceSelect>();
        var innerSelect = resourceSelect.Find("fluent-select");
        innerSelect.Change("TestApp2");
```